core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -205,19 +209,16 @@ private[data] trait OneAndLowPriority1 extends OneAndLowPriority0 {
 }
 
 private[data] trait OneAndLowPriority2 extends OneAndLowPriority1 {
-  implicit def catsDataTraverseForOneAnd[F[_]](implicit F: Traverse[F]): Traverse[OneAnd[F, ?]] =
-    new Traverse[OneAnd[F, ?]] {
-      def traverse[G[_], A, B](fa: OneAnd[F, A])(f: (A) => G[B])(implicit G: Applicative[G]): G[OneAnd[F, B]] = {
-        G.map2Eval(f(fa.head), Always(F.traverse(fa.tail)(f)))(OneAnd(_, _)).value
-      }
+  implicit def catsDataTraverse1ForOneAnd[F[_]](implicit F: Traverse[F], F2: MonadCombine[F]): Traverse1[OneAnd[F, ?]] =
```

Comment:
Can we override `traverse` using the implementation originally used in `Traverse`?
We probably want to override the `traverse` for `NonEmptyVector` as well using the previous implementation, come to think of it.

This implementation using `MonadCombine` also means that you cannot get a `Traverse[OneAnd[F, ?]]` instance anymore if `F` has a `Traverse` but no `MonadCombine`.